### ISSUE TITLE
Fast track account reset?

### ISSUE THREAD

**Issue by the-byte-bender** [keybase/keybase-issues#4152]
Hello, guys!

I'm new to Keybase and didn't know what paper keys were, thus
I believed that all I needed was my password.
Anyway, I lost keybase when I migrated from Windows to Linux.
I reset my account because I wanted to at least keep my username, but now I have to wait for 5 days.
Since I would really need to have things done before five days, I was wondering if there was a way to expedite that.

Thanks!

**Comment by GwynethLlewelyn** [comment]
Well, I guess that the answer is 'no' lol — a month later, did you manage to log back in?

In any case, don't trust only on the paper keys. Make sure that you don't stick to just one device — have at least two, such as your mobile phone. That way, if you ever lose Keybase on one device, you will still have everything on the other one, and will be able to authenticate a new device from any one of the other ones!

**Comment by the-byte-bender** [comment]
> Well, I guess that the answer is 'no' lol — a month later, did you manage to log back in?
> 
> In any case, don't trust only on the paper keys. Make sure that you don't stick to just one device — have at least two, such as your mobile phone. That way, if you ever lose Keybase on one device, you will still have everything on the other one, and will be able to authenticate a new device from any one of the other ones!

Yes I had to wait  the needed 5 days which actually seemed to be 6. I have authenticated my account now with  my phone and my laptop and I backed up a paper key in the most unsafe places possible just to make sure I get hacked as soon as possible, hopefully I'm important enough for people to bother ruining my day

**Comment by ragecc** [comment]
Their recovery system sucks. I have been locked out for months and cant even get anyone to respond period.
